testnets/babylontestnet/assetlist.json

```diff
@@ -267,6 +267,32 @@
         }
       ],
       "type_asset": "ics20"
+    },
+    {
+      "base": "ebbn",
+      "name": "Babylon Testnet Escher LST",
+      "address": "bbn1s5qwgvzzvs5h2wurz7mjwmc4n650g3207caddlz35fay8cl5ay6ss86ejy",
+      "symbol": "eBABY",
```

Comment:
for LSTs please define the trace. I assume something like:
```
"traces": [
  {
    "type": "liquid-stake",
    "counterparty": {
      "chain_name": "babylontestnet",
      "base_denom": "ubbn"
    },
    "provider": "Escher"
  }
]
```

Reply:
I see. Yes, exactly that. Fixed 6a3d85c1e9f4ed5c6822a01bea38e604c083e5d1